src/lib.rs

```diff
@@ -1048,10 +1108,41 @@ impl<'a> EbpfVmFixedMbuff<'a> {
     /// let res = vm.execute_program(mem).unwrap();
     /// assert_eq!(res, 3);
     /// ```
-    pub fn register_helper(&mut self, key: u32, function: fn (u64, u64, u64, u64, u64) -> u64) -> Result<(), Error> {
+    pub fn register_helper(&mut self, key: u32, function: ebpf::HelperFunction) -> Result<(), Error> {
         self.parent.register_helper(key, function)
     }
 
+    /// Register a user-defined helper function in order to use it later from within
```

Comment:
This big comment is duplicated multiple times, which means at least N-1 versions of it will be out of date in the near future.  Can we de-dup?

Reply:
LOL, yeah, this is a precedent set by the original owner.  Its possible there is an idiomatic way to better document this.